eng/Version.Details.xml

```diff
@@ -51,6 +51,18 @@
       <Uri>https://github.com/mono/linker</Uri>
       <Sha>2d3153b356d840819e19a5b2c4c16ada20ecaa53</Sha>
     </Dependency>
+    <Dependency Name="System.CodeDom" Version="4.6.0-preview7.19281.6">
```

Comment:
@nguerrera I think these should be tied to the Microsoft.NETCore.App dependency using this attribute:

CoherentParentDependency="Microsoft.NETCore.App"

This would ensure that those corefx dependencies never get ahead of the netcore.app that is referenced in toolset, which is the typical intended behavior.